M2/cmake/FindFFI.cmake

```diff
@@ -1,3 +1,4 @@
+################### This file is made obsolete by the PR 2968 #######################
```

Comment:
This is an open source script used by [other](https://github.com/llvm/llvm-project/blob/main/llvm/cmake/modules/FindFFI.cmake) projects. I don't see a point in marking it obsolete ...

Reply:
This file is made obsolete for M2, it does not mean it's obsolete in general (although in fact it probably is :-))

Reply:
By the way, LLVM has updated that file few days ago, so M2 copy is obsolete, regardless of this PR :-)